src/components/ButtonPanel.js

```diff
@@ -1,45 +1,52 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-function ButtonPanel() {
+function ButtonPanel(props) {
+  const handleClick = (buttonName) => props.clickHandler(buttonName);
+
   return (
     <>
       <div className="Group-1">
-        <Button buttonName="AC" />
-        <Button buttonName="+/-" />
-        <Button buttonName="%" />
-        <Button buttonName="/" />
+        <Button buttonName="AC" clickHandler={handleClick} />
+        <Button buttonName="+/-" clickHandler={handleClick} />
+        <Button buttonName="%" clickHandler={handleClick} />
+        <Button buttonName="/" clickHandler={handleClick} />
       </div>
```

Comment:
[Optional]
- Instead of writing all the buttons one by one. You can put them into an array and loop through them.
```
Hint:
const group1 = ['AC', '+/-', '%', '÷'];
group1.map(
    curr => {
 .... use per your need
})

``` 

You can use similar technique for another group as well.